circle.yml

```diff
@@ -1,7 +1,10 @@
 machine:
   php:
-    version: 5.4.37
+    version: 5.3.25
 
 test:
   override:
+    # syntax check php files
+    - find ./src -name "*.php" -type f -print0 | xargs -0 -I {} php -l {}
```

Comment:
'compile' step to ensure php 5.3 compatibility in case tests don't have 100% code coverage